### PR TITLE
Ram/jupiter codecept init

### PR DESCRIPTION
**Pull request by ram5607**
https://jira.rax.io/browse/APOLLO-1686
* Adding Codecept to Jupiter pipeline.
[Codecept is configured to run only one scenario. "Verify codecept is loading circle ci"].

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -89,19 +110,23 @@ workflows:
...
+          requires:
+            - build
+
       # All branches must go to staging before going to master - enforced by git branch protection.
       - staging-deploy-ask:
           <<: *not_master
           type: approval
           requires:
-            - build
+            - acceptance
```
Comment (nicko-winner) [inline .circleci/config.yml:122]: I think we want to do this after deploy staging.

Comment (ram5607) [reply .circleci/config.yml]: Yes, updated "acceptance" as a separate job after "deploy-staging". 
And added "deploy-staging" as requires. 

looking for your 
image

File: .circleci/config.yml [hunk 2]
```diff
@@ -42,6 +42,27 @@ jobs:
...
+      - setup_remote_docker:
+          remote: true
+          reusable: true
+          docker_layer_caching: true
+          version: '18.09.3'
+
+      - run:
+          name: Run acceptance tests
+          command: |
+            cd acceptance; docker-compose build && docker-compose up -d
```
Comment (nicko-winner) [inline .circleci/config.yml:63]: I wonder if there would be a noticeable speed benefit to save as a custom image. https://circleci.com/docs/2.0/custom-images/ , according to the docs its a faster processes. We can always add this as an improvement.

Comment (nicko-winner) [reply .circleci/config.yml:63]: With caching the run times are pretty decent:

Comment (ram5607) [reply .circleci/config.yml:63]: yes. thank-you.

**Pull request merged**